src/core/server/pulse/send_pulse.ts

```diff
@@ -65,12 +63,21 @@ export async function sendPulse<Response>(
     // the instructions polling should handle logging for this case, yay for POCs
     return;
   }
+  if (response.status === 200) {
+    for (const sentChannel of channelsToSend) {
+      const channel = channels.get(sentChannel.channel_id);
+      const recordHashes = sentChannel.records.map((record: any) => record.hash).filter(Boolean);
```

Comment:
I wonder why we are relying on `.hash` instead of `.id`

Reply:
just for simplicity, id is not returned in es `_source` so we have to map it there. Also i am think that not everyone wants a hash for their documents so the id can be sometimes randomly generated by es but on other times meaningful like in the newsfeed so a `hash` is an easy way to differentiate.